R/package.R

```diff
@@ -9,8 +9,6 @@
 #' @name cuml
 NULL
 
-.cuml_parsnip_init_done <- FALSE
```

Comment:
This still doesn't make `devtools::load_all()` + `devtools::test()` completely happy because of the unusual way `devtool` handles package loading.

And then after some trial and error, I found out the only reliable way is to do this type of checks instead: `"cuml" %in% parsnip::get_fit(<model type>)$engine` in each of the `register_<model type>()` methods.